cmd/disk-cache-backend.go

```diff
@@ -68,6 +67,8 @@ type cacheMeta struct {
 	Meta map[string]string `json:"meta,omitempty"`
 	// Ranges maps cached range to associated filename.
 	Ranges map[string]string `json:"ranges,omitempty"`
+	// Hits is a counter on the number of times this object has been accessed so far.
+	Hits int `json:"hits,omitempty"`
```

Comment:
should this be trigger hits?

Reply:
this is an internal field in the cache.json and records actual number of accesses so far.